### DISCUSSION TITLE
绝区零开服了吗

### DISCUSSION THREAD

**Discussion by Hongxuli2006** [share121/inter-knot#2034] [其他]
各位刚刚氪金648请问开服了吗。好着急

**Comment by HFOxin** [comment]
还没有开服，这游戏现在看来至少要到11451年4月才能公测了

**Reply by poolofwater** [reply]
真的有必要活到这么臭的时代吗（
<img src="https://img-static.mihoyo.com/communityweb/upload/2bc65c226456bea0398f59387142d6cf.png" alt="帕姆 震惊" width="75" >
